ambry-api/src/main/java/com/github/ambry/config/DiskManagerConfig.java

```diff
@@ -32,11 +32,35 @@ public class DiskManagerConfig {
   @Default("false")
   public final boolean diskManagerEnableSegmentPooling;
 
+  @Config("disk.manager.disk.healthcheck.enabled")
+  @Default("false")
+  public final boolean diskManagerDiskHealthCheckEnabled;
+
+  @Config("disk.manager.disk.healthcheck.interval.seconds")
+  @Default("60")
```

Comment:
is 60 good enuf ?

Reply:
Yeah it was discussed with the team that it would be a good frequency

Reply:
I think 60 is too aggressive. Can you make it 180 ? We do not want the health-check interfering with regular traffic. If there are no issues in the first deployment for two weeks, we can reduce the interval.

Reply:
The media-sre team thought a minute would be good to avoid stale information especially to know if the disks are mounted or not. They were thinking at most 120 in the last meeting with the stakeholders, do you think that wouldn’t be too aggressive or just ramp up from 180 in the coming weeks like you said?

Reply:
I spoke with my team and scheduling the disk health to occur every 3 minutes for now is fine. As long as we scale down to 1 minute if the deployment w/ 3 minutes doesn't causes issues.